x501/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "x501"
+version = "0.1.0" # Also update html_root_url in lib.rs when bumping this
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dev-dependencies]
+hex-literal = "0.3"
+
+[dependencies]
+der = { version = "=0.6.0-pre.0", features = ["derive", "alloc", "oid"], path = "../der" }
```

Comment:
You might include a slightly more complete `Cargo.toml` [looking off one of the other crates as a reference](https://github.com/RustCrypto/formats/blob/master/x509/Cargo.toml).

We generally include license information, links to the repository, keywords, categories, specify the `README.md` name, etc.

Reply:
Oops! I meant to do that before pushing but obviously forgot. Fixed now.